### DISCUSSION TITLE
Adding primary data to JSON file

### DISCUSSION THREAD

**Discussion by MikeDThorpe** [lukePeavey/quotable#47] [Questions]
Hi, fairly new to this but having a go at making my own API similar to quotable but with sport quotes. Has anyone got any suggestions as the quickest way to actually get the data into a json file with all the relevant key/value pairs (i.e id, tags, author). Assuming i've found all the quotes i want to use. I'm hoping there's a quicker, automated way to do it than simply manually typing them into the file. Thanks :)

**Comment by lukePeavey** [comment]
Yes you can write a simple script to extract the data and convert it to JSON. What form is the data in right now ?

**Reply by MikeDThorpe** [reply]
i've made a small json file to test with but the rest of my data is still to be sourced. What form would you recommend and what's the script?
